common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt

```diff
@@ -47,13 +48,24 @@ constructor(
 
         setTypography(typographyType)
 
+        val textColor = if (typedArray.hasValue(R.styleable.DaxTextView_textColor)) {
+            TextColor.from(typedArray.getInt(R.styleable.DaxTextView_textColor, 0))
+        } else {
+            TextColor.Primary
+        }
+        setTextColorStateList(textColor)
+
```

Comment:
Question: Do they have this new attribute defined in Figma so we can check it when implementing it?